controllers/concept.go

```diff
@@ -127,11 +127,11 @@ func (u ConceptController) RetrieveBreakdownStatsBySourceIdAndCohortId(c *gin.Co
 	c.Abort()
 }
 
-func (u ConceptController) RetrieveBreakdownStatsBySourceIdAndCohortIdAndConceptIds(c *gin.Context) {
-	sourceId, cohortId, conceptIds, err1 := utils.ParseSourceIdAndCohortIdAndConceptIds(c)
+func (u ConceptController) RetrieveBreakdownStatsBySourceIdAndCohortIdAndVariables(c *gin.Context) {
+	sourceId, cohortId, conceptIds, cohortPairs, err1 := utils.ParseSourceIdAndCohortIdAndVariablesList(c)
 	breakdownConceptId, err2 := utils.ParseBigNumericArg(c, "breakdownconceptid")
 	if err1 == nil && err2 == nil {
```

Comment:
just curious why we follow a slightly different pattern here compared to the other concept functions. Specifically we usually check for `err1 != nil { log error, abort} ... logic` instead of  checking `err1 == nil {logic} ..... log error, abort`

Reply:
not sure...perhaps a copy+paste of another one... we can easily turn it around. Let me do that.